docs/java-rest/high-level/indices/reload_analyzers.asciidoc

```diff
@@ -0,0 +1,45 @@
+--
+:api: reload-analyzers
+:request: ReloadAnalyzersRequest
+:response: ReloadAnalyzersResponse
+--
+
+[id="{upid}-{api}"]
+=== Reload Search Analyzers API
+
+[id="{upid}-{api}-request"]
+==== Reload Search Analyzers Request
+
+An +{request}+ requires an `index` argument:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests-file}[{api}-request]
+--------------------------------------------------
+<1> The index to reload
+
+==== Optional arguments
+The following arguments can optionally be provided:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests-file}[{api}-request-indicesOptions]
+--------------------------------------------------
+<1> Setting `IndicesOptions` controls how unavailable indices are resolved and
+how wildcard expressions are expanded
+
+include::../execution.asciidoc[]
+
+[id="{upid}-{api}-response"]
+==== Reload Search Analyzers Response
+
+The returned +{response}+ allows to retrieve information about the
+executed operation as follows:
+
+["source","java",subs="attributes,callouts,macros"]
+--------------------------------------------------
+include-tagged::{doc-tests-file}[{api}-response]
+--------------------------------------------------
+<1> Shard statistics including total number of shards the request was executed on
```

Comment:
This is not entirely related to this pr but I find it confusing that we return a `Shards` object in the response even though the action is executed at the node level. I don't have a good way to fix this but we should at least document here that the statistics are per node/index and not per shards ?

Reply:
I agree, this is not ideal. I'll look into the options of either documenting it better, change the naming of the object only for the java api level or whether maybe it makes sense change or even leave out the whole "shard: { ... }". That would have to be done further down the calls stack though.

Reply:
I looked into the options, and it would be possible to remove the "shard: { ... }" element from the REST response completely even though TransportReloadAnalyzersAction would still be able to extend BroadcastResponse, but we could overwrite "toXContent" without outputing broadcast shards header. Then again, potential failure exceptions are currently also written inside the "shards" header, replacing that toXContent rendering and parsing might turn out to be a bigger change than intended.
Changing the accessor or object name would also involve either not being able to extend BroadcastResponse on the HLRC client side (this would mean more custom parsing code, other getters etc... for failures), so I'll push an update to the docs for now. If you want to discuss the other options again let me know.